.github/workflows/build-ui.yml

```
@@ -38,27 +38,25 @@ jobs:
       working-directory: ui
       run: npm run build
 
-    - name: Commit changes to PR
-      if: github.event_name == 'pull_request'
+    - name: Check for changes
+      id: check_changes
       run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:53: Double quote to prevent globbing and word splitting

(shellcheck)